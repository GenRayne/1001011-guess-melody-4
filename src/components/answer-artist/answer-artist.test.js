import React from 'react';
import renderer from 'react-test-renderer';
import AnswerArtist from './answer-artist.jsx';

const answer = {
  id: `1`,
  picture: `https://api.adorable.io/avatars/128/10`,
  artist: `John Snow`,
};

describe(`render AnswerArtist`, () => {
  it(`renders AnswerArtist`, () => {
    const tree = renderer
      .create(
          <AnswerArtist
            answer={answer}
            onAnswer={() => {}}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
