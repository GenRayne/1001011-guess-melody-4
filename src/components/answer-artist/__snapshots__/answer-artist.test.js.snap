// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render AnswerArtist renders AnswerArtist 1`] = `
<div
  className="artist"
>
  <input
    className="artist__input visually-hidden"
    id="answer-1"
    name="answer"
    onChange={[Function]}
    type="radio"
    value="artist-1"
  />
  <label
    className="artist__name"
    htmlFor="answer-1"
  >
    <img
      alt="John Snow"
      className="artist__picture"
      src="https://api.adorable.io/avatars/128/10"
    />
    John Snow
  </label>
</div>
`;
